### PR TITLE
code sample WordCount

### PR DESCRIPTION
**Pull request by baaastijn**
code for Data Processing documentation

### CODE REVIEW

File: python_wordCount/wordcount.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+# Count and group word frequencies on the column Lyrics, when splitted by space comma
+data.withColumn('word', f.explode(f.split(f.col('Lyrics'), ' '))) \
+  .groupBy('word') \
+  .count() \
+  .sort('count', ascending=False) \
+  .show()
+
+# To remove stop words (like "I", "The", ...), we need to provide arrays of words, not strings. Here we use APache Spark Tokenizer to do so.
```
Comment (Pirionfr) [inline python_wordCount/wordcount.py:30]: PEP8: E501 line to long (140 > 120 characters)

File: python_wordCount/wordcount.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import sys
```
Comment (Pirionfr) [inline python_wordCount/wordcount.py:1]: unused import

File: python_wordCount/wordcount.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import sys
+
+from operator import add
```
Comment (Pirionfr) [inline python_wordCount/wordcount.py:3]: unused import

File: python_wordCount/readme.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+## WordCount
+
+This code is from the OVHcloud documentation, in the section *Tutorials* of Data Processing:  https://docs.ovh.com/gb/en/data-processing/ and is explained there.
```
Comment (Pirionfr) [inline python_wordCount/readme.md:3]: can you put the real link ?

File: python_wordCount/wordcount.py (hunk 1 above)
Comment (baaastijn) [reply python_wordCount/wordcount.py]: done

**Pull request merged**